demos/demos_by_use_case/bio/cpu/chemical_mappings_before_gpu.ipynb

```diff
@@ -0,0 +1,562 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "view-in-github",
+        "colab_type": "text"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/github/dcolinmorgan/grph/blob/main/generic_chemical_mappings.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "dzmJAwfiAi6k"
+      },
+      "source": [
+        "# Mapping Chemical Space with UMAP (adapted from [blog post](https://blog.reverielabs.com/mapping-chemical-space-with-umap))\n",
```

Comment:
* drop use of non-graphistry umap
* instead of 2 notebooks cpu/gpu, try to do a very short cpu umap cell viz+timing vs gpu umap cell viz+timing